src/components/track/track.css

```diff
@@ -0,0 +1,27 @@
+.track {
+  display: grid;
+  grid-template-columns: 50px 1fr 40px;
+  text-align: left;
+  width: 375px;
```

Comment:
I'd probably use width:100% and then fix some of the other spacing (since setting width to 100% would destroy some of the spacing), but right now you have a scroll bar on the small mobile view.
🤗

Reply:
yes, that´s a good suggestion. I was thinking of changing it to 100%, when i put it together on the page. 😊